apps/settings/src/components/DeclarativeSettings/DeclarativeSection.vue

```diff
@@ -0,0 +1,266 @@
+<template>
+	<NcSettingsSection
+		class="declarative-settings-section"
+		:name="t(formApp, form.title)"
+		:description="t(formApp, form.description)"
+		:doc-url="form.doc_url || ''">
+		<div v-for="formField in formFields"
+			 :key="formField.id"
+			 class="declarative-form-field"
+			:aria-label="t('settings', '{app} declarative setting field: {name}', { app: formApp, name: t(formApp, formField.title) })"
```

Comment:
```suggestion
			:aria-label="t('settings', '{app}\'s declarative setting field: {name}', { app: formApp, name: t(formApp, formField.title) })"
```

Also the name translation part needs an explanation comment about why it works with forms registered by class and by event. And for the event case, why it works with Php apps and exApps.